### PR TITLE
BK Precision 9130B Instrument

### PR DESCRIPTION
**Pull request by dennisfeng2**
Added the beginning of the new instrument: BK Precision 9130B

### CODE REVIEW

**Comment by dennisfeng2** [comment]
I've noticed that there are some failed tests/checks with this pull request, but after taking a look at the error messages, it's still unclear to me whether these issues are coming from bugs in the pull request itself. For example, some of the errors involve import errors of VXI11Adapter and HP34410A, which my code doesn't seem to touch at all. Could someone point out to me why these errors are occurring, or anything I can fix from my end, so we can make progress on this PR?

**Comment by dennisfeng2** [comment]
Okay, merged with latest master and added the RST documentation. Could you take a look?

File: docs/api/instruments/bkprecision/bkprecision9130b.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#########################
```
Comment (cjermain) [inline docs/api/instruments/bkprecision/bkprecision9130b.rst]: The header sections (i.e. `####`) should match the title in length.
```
##################################
BK Precision 9130B DC Power Supply
##################################
```

File: pymeasure/instruments/bkprecision/bkprecision9130b.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import strict_discrete_set, truncated_range
+
+CHANNEL_NUMS = [1, 2, 3]
+
+
+class BKPrecision9130B(Instrument):
+    """ Represents the BK Precision 9130B DC Power Supply interface for interacting with the instrument. """
+
+    current = Instrument.control('MEASure:SCALar:CURRent:DC?', 'SOURce:CURRent:LEVel:IMMediate:AMPLitude %g',
```
Comment (cjermain) [inline pymeasure/instruments/bkprecision/bkprecision9130b.py]: Please update the style with the comments in #198.

File: pymeasure/instruments/bkprecision/bkprecision9130b.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        )
+
+    @property
+    def voltage(self):
+        """Floating point property used to control voltage of the selected channel."""
+        return float(self.ask("MEASure:SCALar:VOLTage:DC?"))
+
+    @voltage.setter
+    def voltage(self, level):
+        if self.channel == 3:
```
Comment (cjermain) [inline pymeasure/instruments/bkprecision/bkprecision9130b.py]: One way to shorten this `if` statement, while keeping readability, is to write it as:
```
range = [0, 5] if self.channel == 3 else [0, 30]
new_level = truncated_range(level, range)
```

File: docs/api/instruments/bkprecision/bkprecision9130b.rst (hunk 1 above)
Comment (dennisfeng2) [reply docs/api/instruments/bkprecision/bkprecision9130b.rst]: done

File: pymeasure/instruments/bkprecision/bkprecision9130b.py (hunk 2 above)
Comment (dennisfeng2) [reply pymeasure/instruments/bkprecision/bkprecision9130b.py]: done

File: pymeasure/instruments/bkprecision/bkprecision9130b.py (hunk 3 above)
Comment (dennisfeng2) [reply pymeasure/instruments/bkprecision/bkprecision9130b.py]: done

**Comment by dennisfeng2** [comment]
Hi, apologies that this is a bit of an old PR, but I finally got around to addressing the comments, and I'd like to push this forward. Are there any more code changes you'd like me to make, before we can merge it?

**Comment by dennisfeng2** [comment]
Bumping this, since it's probably relatively easy to review and merge (b/c it already has been code reviewed once)

**Comment by bilderbuchi** [comment]
Looking good, thank you! It's never too late, we are also sometimes slow to get around to stuff. :-)

**Comment by dennischenfeng** [comment]
Nice, no worries! Thank you :)

**Comment by dennisfeng2** [comment]
(Oops, I meant to thank you through this account rather than my other github account above) 
Thanks for merging!

**Pull request merged**